client/app/bundles/course/assessment/question/scribing/actions/scribingQuestionActionCreators.jsx

```diff
@@ -0,0 +1,110 @@
+import _ from 'lodash';
+import CourseAPI from 'api/course';
+import { getCourseId, getAssessmentId, getScribingId } from 'lib/helpers/url-helpers';
+import { submit, SubmissionError } from 'redux-form';
+import actionTypes, { formNames } from '../constants';
+
+export function submitForm() {
+  return (dispatch) => {
+    dispatch(submit(formNames.SCRIBING_QUESTION));
+  };
+}
+
+export function fetchScribingQuestion(scribingId) {
+  return (dispatch) => {
+    dispatch({ type: actionTypes.FETCH_SCRIBING_QUESTION_REQUEST });
+    return CourseAPI.scribing.scribings.fetch(scribingId)
+      .then((response) => {
+        dispatch({
+          scribingId: CourseAPI.scribing.scribings.getScribingId(),
+          type: actionTypes.FETCH_SCRIBING_QUESTION_SUCCESS,
+          data: response.data,
+        });
+      })
+      .catch((error) => {
+        dispatch({ type: actionTypes.FETCH_SCRIBING_QUESTION_FAILURE });
+        if (error.response && error.response.data) {
+          throw new SubmissionError(error.response.data.errors);
+        }
+      });
+  };
+}
+
+// Helper function to convert array of skills to array of skill_ids
+function getSkillIdsFromSkills(skills) {
+  const ids = _.map(skills, skill => skill.id);
+  return (ids.length > 0) ? ids : [''];
```

Comment:
It's weird that you return an array of one item if the length is 0. Anyway, it's better to check length before doing the mapping and you can shorten into one line:

`return skills.length > 0 ? _.map(skills, 'id') : [''];`